freqtrade/templates/FreqaiHybridExampleStrategy.py

```diff
@@ -0,0 +1,262 @@
+        dataframe["bb_percent"] = (
+            (dataframe["close"] - dataframe["bb_lowerband"]) /
+            (dataframe["bb_upperband"] - dataframe["bb_lowerband"])
+        )
+        dataframe["bb_width"] = (
+            (dataframe["bb_upperband"] - dataframe["bb_lowerband"]) / dataframe["bb_middleband"]
+        )
+
+        # TEMA - Triple Exponential Moving Average
+        dataframe['tema'] = ta.TEMA(dataframe, timeperiod=9)
+
+        return dataframe
+
+    def populate_entry_trend(self, df: DataFrame, metadata: dict) -> DataFrame:
+
+        df.loc[
+            (
+                # Signal: RSI crosses above 30
+                (qtpylib.crossed_above(df['rsi'], self.buy_rsi.value)) &
+                (df['tema'] <= df['bb_middleband']) &  # Guard: tema below BB middle
+                (df['tema'] > df['tema'].shift(1)) &  # Guard: tema is raising
+                (df['volume'] > 0) &  # Make sure Volume is not 0
+                (df['do_predict'] == 1) &  # Make sure Freqai is confident in the prediction
+                # Only enter trade if Freqai thinks the trend is in this direction
+                (df['&s-up_or_down'] == 'up')
+            ),
+            'enter_long'] = 1
+
+        df.loc[
+            (
+                # Signal: RSI crosses above 70
+                (qtpylib.crossed_above(df['rsi'], self.short_rsi.value)) &
+                (df['tema'] > df['bb_middleband']) &  # Guard: tema above BB middle
+                (df['tema'] < df['tema'].shift(1)) &  # Guard: tema is falling
+                (df['volume'] > 0) &  # Make sure Volume is not 0
+                (df['do_predict'] == 1) &  # Make sure Freqai is confident in the prediction
+                # Only enter trade if Freqai thinks the trend is in this direction
+                (df['&s-up_or_down'] == 'down')
+            ),
+            'enter_short'] = 1
+
+        return df
+
+    def populate_exit_trend(self, df: DataFrame, metadata: dict) -> DataFrame:
+
+        df.loc[
+            (
+                # Signal: RSI crosses above 70
+                (qtpylib.crossed_above(df['rsi'], self.sell_rsi.value)) &
+                (df['tema'] > df['bb_middleband']) &  # Guard: tema above BB middle
+                (df['tema'] < df['tema'].shift(1)) &  # Guard: tema is falling
+                (df['volume'] > 0)  # Make sure Volume is not 0
+            ),
+
+            'exit_long'] = 1
+
+        df.loc[
+            (
+                # Signal: RSI crosses above 30
+                (qtpylib.crossed_above(df['rsi'], self.exit_short_rsi.value)) &
+                # Guard: tema below BB middle
+                (df['tema'] <= df['bb_middleband']) &
+                (df['tema'] > df['tema'].shift(1)) &  # Guard: tema is raising
+                (df['volume'] > 0)  # Make sure Volume is not 0
+            ),
+            'exit_short'] = 1
+
+        return df
+
+    def get_ticker_indicator(self):
+        return int(self.config["timeframe"][:-1])
```

Comment:
Can we remove this?
It doesn't seem to be used - and it's faulty code.

it'll return `1` - no matter if the timeframe is 1m, 1h or 1d ... so it'll not work properly once you start changing timeframes around

Reply:
Done